### ISSUE TITLE
App stops bootstrapping when @angular/router version changed to 3.4.8

### ISSUE THREAD

**Issue by pdhar-tibco** [angular/angular#14686]
**I'm submitting a ...**  (check one with "x")
```
[ X] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
https://plnkr.co/edit/RSCvxk?p=preview
Using @angular/router version 3.4.8 in config.js line 24 the app does not bootstrap

**Expected behavior**
The app should bootstrap.

**Minimal reproduction of the problem with instructions**
Change the version of @angular/router in config.js line 24 to 3.4.7 to see the app working.

https://plnkr.co/edit/RSCvxk?p=preview
**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**
https://plnkr.co/edit/RSCvxk?p=preview

* **Angular version:** 2.0.X
2.4.8

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

* **Node (for AoT issues):** `node --version` =

**Comment by DzmitryShylovich** [comment]
dup https://github.com/angular/angular/issues/14588

**Comment by DzmitryShylovich** [comment]
@pdhar-tibco already fixed in 2.4.9
close the issue please. thanks
